### PR TITLE
Update the `az.create-aks` task

### PR DESCRIPTION
**Pull request by FlorentClarret**
What does this PR do?
---------------------

- Update the `az.create-aks` to create clusters in agent sandbox/qa
- We do not add the `az.create-aks` to the integration tests in this CI, it takes only around 8 minutes but we do not have a lot of resources on Azure

Some semi-related tasks to: 
- Update the documentation to remove explicit mention of `aws-vault exec` commands, not needed anymore since https://github.com/DataDog/test-infra-definitions/pull/917
- Update the readme with the new commands from https://github.com/DataDog/test-infra-definitions/pull/959
- Update the configuration file used in the CI to configure the account used to run the tests
- Update the example config files

Which scenarios this will impact?
-------------------

Motivation
----------

Allow users to create AKS in our new sandbox

Additional Notes
----------------

- I had to follow https://learn.microsoft.com/en-us/azure/aks/use-pod-sandboxing#install-the-aks-preview-azure-cli-extension to make it work in our new sandbox, I'll document it.
- We used to have `ssh-ed25519` keys [here](https://github.com/DataDog/test-infra-definitions/blob/680f0b000d20fcaa8617da9a83c741b68d214c86/.gitlab-ci.yml#L70-L71). They do not work with AKS and we explicitly need `ssh-rsa` keys so I created a new pair and pushed it

### CODE REVIEW

File: integration-tests/invoke_test.go [hunk 1]
```diff
@@ -55,17 +55,22 @@ func TestInvokes(t *testing.T) {
 		testAzureInvokeVM(t, tmpConfigFile, *workingDir)
 	})
 
+	t.Run("az.create-aks", func(t *testing.T) {
```
Comment (pducolin) [inline integration-tests/invoke_test.go:58]: **🔨 warning**
We have limited aks resources and it takes usually very long to create an aks cluster, similarly to eks. I would rather have it as a manual test

File: README.md [hunk 2]
```diff
@@ -140,21 +150,21 @@ pulumi stack rm <your_name>-ecs-test
...
-aws-vault exec sso-agent-sandbox-account-admin -- pulumi up -c scenario=aws/ecs -c ddinfra:aws/defaultKeyPairName=<your_exisiting_aws_keypair_name> -c ddinfra:env=aws/agent-sandbox -c ddagent:apiKey=$DD_API_KEY -s <your_name>-ecs
+pulumi up -c scenario=aws/ecs -c ddinfra:aws/defaultKeyPairName=<your_exisiting_aws_keypair_name> -c ddinfra:env=aws/agent-sandbox -c ddagent:apiKey=$DD_API_KEY -s <your_name>-ecs
 ```
 
 ## Quick start: Create an EKS (Linux/Windows) + Fargate (Linux) Cluster + Agent (Helm)
 
 ```
 # You need to have a DD APIKey AND APPKey in variable DD_API_KEY / DD_APP_KEY
-aws-vault exec sso-agent-sandbox-account-admin -- pulumi up -c scenario=aws/eks -c ddinfra:aws/defaultKeyPairName=<your_exisiting_aws_keypair_name> -c ddinfra:env=aws/agent-sandbox -c ddagent:apiKey=$DD_API_KEY -c ddagent:appKey=$DD_APP_KEY -s <your_name>-eks
+pulumi up -c scenario=aws/eks -c ddinfra:aws/defaultKeyPairName=<your_exisiting_aws_keypair_name> -c ddinfra:env=aws/agent-sandbox -c ddagent:apiKey=$DD_API_KEY -c ddagent:appKey=$DD_APP_KEY -s <your_name>-eks
```
Comment (pducolin) [inline README.md:167]: **👏 praise**

File: integration-tests/invoke_test.go [hunk 3]
```diff
@@ -76,7 +76,7 @@ func testAzureInvokeVM(t *testing.T, tmpConfigFile string, workingDirectory stri
 
 	stackName := fmt.Sprintf("az-invoke-vm-%s", os.Getenv("CI_PIPELINE_ID"))
 	t.Log("creating vm")
-	createCmd := exec.Command("invoke", "az.create-vm", "--no-interactive", "--stack-name", stackName, "--config-path", tmpConfigFile, "--account", "agent-qa")
```
Comment (pducolin) [inline integration-tests/invoke_test.go:79]: **❓ question**
It doesn't work with `agent-qa`? Or we are setting it at setup?

**Review by pducolin — APPROVED** [review]

File: integration-tests/invoke_test.go (hunk 3 above)
Comment (FlorentClarret) [reply integration-tests/invoke_test.go:79]: I moved it to the config file: https://github.com/DataDog/test-infra-definitions/pull/967/files#diff-5b0d7f0d4cd5bb98edb0f1e6b5ac0747cae4be958137b00d9785e4863a90acdcR12 sorry it was not super explicit. We already do the same thing with AWS so I just centralised everything

**Pull request merged**